### PR TITLE
Fixes #1139 - Add Frame.select method 

### LINKED ISSUE #1139: Missing ElementHandle.select; impossible (?) to make selection in a frame
There is no `frame.select` equivalent of the [`page.select`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageselectselector-values). As far as I can tell, there is no way to select an option within an iframe.

### PR DESCRIPTION
**Pull request by ghost**
Added Frame.select method for issue #1139

### CODE REVIEW

**Comment by infosuvratjain** [comment]
I signed it!

**Comment by ghost** [comment]
@googlebot  I signed it!

File: lib/FrameManager.js [hunk 1]
```diff
@@ -405,7 +406,28 @@ class Frame {
       document.head.appendChild(style);
     }
   }
+  /**
+  *@param {string} selector
```
Comment (aslushnikov) [inline lib/FrameManager.js:410]: nit: formatting is broken here

File: lib/FrameManager.js [hunk 2]
```diff
@@ -405,7 +406,28 @@ class Frame {
       document.head.appendChild(style);
     }
   }
+  /**
```
Comment (aslushnikov) [inline lib/FrameManager.js:409]: nit: please add a new line prior to the comment

File: lib/FrameManager.js [hunk 3]
```diff
@@ -405,7 +406,28 @@ class Frame {
...
+  *@param {string} selector
+  *@param {!Array<string>} values
+   */
+
```
Comment (aslushnikov) [inline lib/FrameManager.js:413]: nit: drop the newline

File: lib/FrameManager.js [hunk 4]
```diff
@@ -405,7 +406,28 @@ class Frame {
...
+  async select(selector, ...values){
```
Comment (aslushnikov) [inline lib/FrameManager.js:414]: `page.select` should now use main frame's select:

```js
class Page {
  // ...
  async select(selector, ...values) {
    return this.mainFrame().select(selector, ...values);
  }
  // ...
}
```
Use `page.mainFrame().select()`

File: test/test.js [hunk 5]
```diff
@@ -756,6 +756,58 @@ describe('Page', function() {
     }));
   });
 
+  describe('Frame.select', function() {
```
Comment (aslushnikov) [inline test/test.js:759]: ... and these tests won't be needed then; the `page.select` tests will cover the whole thing.

**Review by aslushnikov — COMMENTED** [review]
Thanks!
Please note: there's #1099 that touches the same code. Hopefully, it'll land soon, so please be prepared to rebase atop of it.

**Comment by suvratjain1995** [comment]
@aslushnikov  Working on the changes as the fixes that you mentioned have been commited.

**Comment by aslushnikov** [comment]
Closing this in favor of #1243

**Pull request closed without merging**